app/models/deploy_service.rb

```diff
@@ -131,7 +131,9 @@ def notify_outbound_webhooks(deploy)
     deploy.stage.outbound_webhooks.each { |webhook| webhook.deliver(deploy) }
   end
 
-  def send_sse_deploy_update(type)
-    SseRailsEngine.send_event('deploys', type: type)
+  def send_deploy_update(finished = false)
```

Comment:
Kinda funky signature / method use combo passing in a symbol and defaulting to a boolean. Maybe make it a keyword arg `finished: false`? Or else rename it to `exclude_self` or `decrement` to make things more clear?